routing/payment_lifecycle.go

```diff
@@ -24,14 +26,85 @@ type paymentLifecycle struct {
 	currentHeight int32
 }
 
+// paymentState uses the passed payment to find the latest information we need
+// to act on every iteration of the payment loop.
+func (p *paymentLifecycle) paymentState(payment *channeldb.MPPayment) (
+	uint32, lnwire.MilliSatoshi, lnwire.MilliSatoshi, bool, error) {
```

Comment:
returning a struct with named fields wouldn't be bad

Reply:
done